docs/connector/spark/hive.rst

```diff
@@ -106,4 +101,29 @@ Taking ``DROP NAMESPACE`` as a example,
 
    DROP NAMESPACE hive_catalog.ns;
 
-.. _Apache Spark: https://spark.apache.org/
\ No newline at end of file
+Advanced Usages
+***************
+
+Though KSCH is a pure Spark DataSource V2 connector which isn't coupled with Kyuubi deployment, due to the
```

Comment:
should we suggest users to set `spark.sql.catalog.hive_catalog.spark.sql.hive.metastore.jars` explicitly to bypass some problems if using `spark-sql`?

Reply:
No, I would not recommend using `IsolatedClassloader`, for any purpose. Actually, if you read the code of `spark-sql`, you may find that it claims it does not support `IsolatedClassloader`, I'm not sure why such a combination works, maybe another side-effect of a bug.